tf2_ros/include/tf2_ros/message_filter.h

```diff
@@ -377,8 +393,8 @@ class MessageFilter : public MessageFilterBase, public message_filters::SimpleFi
               info.handles.push_back(next_handle_index_++);
             }
           }
-          catch (...) {
-            // never transformable
+          catch (std::exception & e)  {
```

Comment:
Could we catch a better exception than this?

Reply:
See my comment above.